storymap_templates/gdh_default_storymap_template.yaml

```diff
@@ -1,14 +1,180 @@
+
+panels:
+  # Background
+  - type: text
+    style: heading
+    text: "1. Background"
+
+  - type: text
+    style: paragraph
+    text: *Background
+
+  - type: separator
+
+  # Area_of_Interest
+  - type: text
+    style: heading
+    text: "2. Area of Interest"
+
+  - type: map
+    item_id: "{negotiated_design_item_id}"
+    caption: "Figure 1: Design Area"
+
+  - type: text
+    style: paragraph
+    text: *Area_of_Interest
+  
+  - type: separator
+
+  # Broader_Objectives
+  - type: text
+    style: heading
+    text: "3. Broader objectives"
+
+  - type: text
+    style: paragraph
+    text: *Objectives
+
+  - type: separator
+
+  # Compiling_Prior_Work
+  - type: text
+    style: heading
+    text: "4. Compiling prior work"
+
+  - type: text
+    style: paragraph
+    text: *Compiling_Prior_Work
+
+  - type: separator
+
+  # Geosurvey
+  - type: text
+    style: heading
+    text: "5. Geosurvey"
+
+  - type: text
+    style: paragraph
+    text: *Geosurvey
+
+  - type: image
+    url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/0eed20c97d74c71bc9f937b5bea3a8b2f8412a6b.jpeg"
+    caption: "Graph"
+
+  - type: text
+    style: paragraph
+    text: *Geosurvey2
+
+  - type: table
+    headers:
+      - "Improvement Suggestion"
+      - "Comment"
+    rows:
+      - ["Better infrastructure", "Improved road networks"]
+      - ["Increase greenery", "Plant more trees in urban areas"]
+      - ["Flood prevention", "Build retention ponds"]
+
+  - type: separator
+
+  # Workshop_Schedule
+  - type: text
+    style: heading
+    text: "6. Workshop Schedule"
+
+  - type: text
+    style: paragraph
+    text: *Workshop_Schedule
+
+  - type: image
+    url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/ef0664e002d38cbbad258bbcdfdfcdc7cf985a6e.jpg"
+    caption: "Original Source: Emil Lanfranchi and Francesco Fonzino"
+
+  - type: separator
+
+  # Participants_of_the_workshop
+  - type: text
+    style: heading
+    text: "7. Participants of the workshop"
+
+  - type: text
+    style: paragraph
+    text: *Participants_of_the_workshop
+
+  - type: gallery
+    images:
+      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/b9e8c9bd2e9fc7eaf6c3280335ff158cc68be7c7.jpg"
+        caption: "Photo 1"
+      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/27979335c966086ea51b2e76d0a6cc799ea7ba81.jpg"
+        caption: "Photo 2"
+      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/d38d45ad6bb1af098903efae70de63da018d7071.jpeg"
+        caption: "Photo 3"
+      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/7af77de5e8efb689e8e94ca32c33c3bfffc3e275.jpg"
+        caption: "Photo 4"
+    caption: "Photos"
+    
+  - type: text
+    style: heading
+    text: "Acknowledgements"
+
+  - type: text
+    style: paragraph
+    text: "We appreciate the contributions of all team members."
```

Comment:
Request to add a concluding section to have acknowledgements e.g. see the last section here: https://storymaps.arcgis.com/stories/f20cd688308e4148916b647466557d30, it should have some text and logos

Reply:
Added more contents